### PR TITLE
Allow validator groups to update their commission

### LINKED ISSUE #1374: Users should be able to set and query the commission rate on groups
### Expected Behavior

Add function to set commission for a group on `Validators.sol`

Users should be able to query the group commission rate and, if they are the owner of the group, set the commission rate through the CLI.

### Current Behavior

Users cannot  query or set commission rates.

### PR DESCRIPTION
**Pull request by asaj**
### Description

This PR adds a function to allow validator groups to update their commission, as well as a corresponding command in celocli.

### Tested

Contract and contractkit unit tests.

### Other changes
- Remove out of date CLI example

### Related issues

- Fixes #1374 

### Backwards compatibility

Technically yes, though the smart contract was changed.

### CODE REVIEW

File: packages/contractkit/src/wrappers/Validators.test.ts [hunk 1]
```diff
@@ -86,6 +86,16 @@ testWithGanache('Validators Wrapper', (web3) => {
...
   })
 
+  test('SBAT updateCommission', async () => {
+    const groupAccount = accounts[0]
+    await setupGroup(groupAccount)
+    await (await validators.updateCommission(new BigNumber(0.2))).sendAndWaitForReceipt({
+      from: groupAccount,
+    })
+    const commission = (await validators.getValidatorGroup(groupAccount)).commission
+    expect(commission.toFixed()).toEqual('0.2')
```
Comment (nategraf) [inline packages/contractkit/src/wrappers/Validators.test.ts]: Could you add an `expect` for commission either in this test or the `setupGroup` function to ensure that the is what you expect before the change (i.e. not `0.2`)

File: packages/protocol/contracts/governance/Validators.sol [hunk 2]
```diff
@@ -651,6 +652,23 @@ contract Validators is
     return true;
   }
 
+  /**
+   * @notice Updates a validator group's commission.
+   * @param commission Fixidity representation of the commission this group receives on epoch
+   *   payments made to its members.
```
Comment (nategraf) [inline packages/protocol/contracts/governance/Validators.sol]: I note about the range may be helpful.
```suggestion
   *   payments made to its members. In the range 0 to 1 inclusive.
```

**Review by nategraf — APPROVED** [review]
Nice!

File: packages/cli/src/commands/validatorgroup/commission.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  static examples = [
+    'commission --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95 --commission 0.1',
+  ]
+
+  async run() {
+    const res = this.parse(ValidatorGroupCommission)
+
+    this.kit.defaultAccount = res.flags.from
+    const validators = await this.kit.contracts.getValidators()
+    const tx = await validators.updateCommission(new BigNumber(res.flags.commission))
```
Comment (mcortesi) [inline packages/cli/src/commands/validatorgroup/commission.ts]: can you add "checks" ?

File: packages/contractkit/src/wrappers/Validators.test.ts (hunk 1 above)
Comment (mcortesi) [inline packages/contractkit/src/wrappers/Validators.test.ts]: there's a  `expect().toEqBigNumber` in contractKit that you can use (no need to import or anything)

**Review by mcortesi — APPROVED** [review]

**Pull request merged**